src/app/modules/sample-providers/components/submission-form/submission-form.component.html

```diff
@@ -1,28 +1,44 @@
 <form [formGroup]="form" (ngSubmit)="onSubmit()">
   <div>
-    <label for="label-name">
-      Name
-    </label>
-    <input  type="text"  id="label-name" formControlName="label_name"/>
+    <app-form-control
```

Comment:
This seems like an unnecessary abstraction. In fact, I would argue that the abstraction is not as semantic and descriptive as a simple html form type.